app/models/market_model.rb

```diff
@@ -0,0 +1,17 @@
+class MarketModel < ActiveRecord::Base
+  include Privacy
+
+  FOUNDATIONS = ["kWh", "kW_connection", "kW_max", "kW_contracted", "kW_flex"]
```

Comment:
I actually think that 'kWh' and 'kW_connection' are going to be the most important (kW_max and kW_contracted are currently only applicable to larger users, say industry). Luckily, I also think these are pretty simple:
* kWh: (load * time) consumed per customer over the whole year.
* kW_connection: a fixed price per year per customer. This is mostly 3x25 Amps. Sometimes 3x80 Amps. We could either make it the responsibility of the user to assign to each end-point of the topology a capacity of either of these two types. The disadvantage of that is that there could be more or fewer households connected to that end-point than the capacity was intended for and the user would have to adjust the topology for every LES.